avocado/core/utils.py

```diff
@@ -34,3 +34,30 @@ def prepend_base_path(value):
         dist = get_distribution('avocado-framework')
         return os.path.join(dist.location, 'avocado', expanded)
     return expanded
+
+
+def system_wide_or_base_path(file_path):
+    """Returns either a system wide path, or one relative to the base.
```

Comment:
Could this be
```python
"""
Returns either a system wide path, or one relative to the base.

...
"""
```
according to the pydocstyle which was used in the nrunner (e.g. Runnable class) module for example?

Reply:
Actually it looks like we were *not* following https://www.python.org/dev/peps/pep-0257/ in this regard, so I'm trying to use this new style whenever I write new docstrings.